docs/bonding/index.adoc

```diff
@@ -39,99 +40,99 @@ complex implementation.
 {extra-collateral}, for a total of {total-collateral} collateralization rate.
 
 [example]
 **Without overcollateralization:** Let 1 BTC be worth $10000, and 1 ETH be
 worth $200. Signers have to put up 50 ETH to back a deposit. Due to market
 conditions, ETH drops 25% to $150, while BTC maintains its value. The 50 ETH
-is worth $7500, meaning the Signers can make a $2500 profit by stealing the
+is worth $7500, meaning the signers can make a $2500 profit by stealing the
 deposit.
 
 [example]
 **With overcollateralization:** Let 1 BTC be worth $10,000, and 1 ETH be worth
 $200. Signers have to put up 75 ETH (150% of 50) to back a deposit. Due to
 market conditions, ETH drops 25% to $150, while BTC maintains its value. The
 75 ETH is worth $11250, which is above the dollar value of BTC meaning the
-Signers should maintain honest behavior since they have more to lose.
+signers should maintain honest behavior since they have more to lose.
 
 In general, total overcollateralization of {total-collateral} (`3/2 * 100%`)
-keeps Signer incentives aligned with the well-being of the system up to a 33%
+keeps signer incentives aligned with the well-being of the system up to a 33%
 drop (`(1 - 2/3) * 100%`) in price of the bonded asset against the deposit's
 asset. Increasing this percentage can increase the robustness of the system,
-at the expense of opportunity cost to the Signers which should be compensated
+at the expense of opportunity cost to the signers which should be compensated
 via fees.
 
-If the value of ETH crosses a security threshold, open _Deposit_ s will enter
-<<pre-liquidation, pre-liquidation>>, followed by <<liquidation, liquidation>>.
-
-// TODO insert a little historical analysis for a decent starting number
-
+If the value of ETH crosses a security threshold, open deposits will enter
+<<pre-liquidation,pre-liquidation>>, followed by <<liquidation,liquidation>>.
 
 === BTC price drop relative to ETH
 
 Since <<signer-fees/index.adoc#signer-fees,signer fees>> are denominated per
 BTC in custody (with overcollateralization factored in), a BTC value drop
-against the bonded asset translates in lower fees for Signers. Note that this
+against the bonded asset translates in lower fees for signers. Note that this
 does not create any issue for TBTC reserves, but it makes the system less
 attractive to signers earning fees on their assets.
 
 Signers SHOULD buy TBTC from the markets in anticipation of such overly
-overcollateralized Deposits and they SHOULD use it to redeem these positions,
-thus reclaiming their ETH liquidity which can be used to back other Deposits. An
-alternative would be to provide Signers with the ability to safely rebalance their
-bonds back to {total-collateral}, however that introduces implementation
-complexities and as a result is not the preferred solution for the initial
-deployment of the mechanism.
+overcollateralized deposits and they SHOULD use it to redeem these positions
```

Comment:
The docs only use the RFC-2119 style keywords in very few places. I assume that is just because the specification is still in progress? Or is that because specification and documentation are synonymous at the moment?

Reply:
A little of both. The original introduction of those keywords was incomplete, which has led to further incompleteness. I don’t currently have time to rewrite the whole spec in the style required to properly apply those keywords consistently, so I’m mostly just trying not to lose them most of the time. 

Frankly, I’d we went full RFC style here, we would probably want to split this into two specs: one that describes a mechanism for transferring UTXO custody to an Ethereum smart contract, and one to describe a mechanism for minting a fungible ERC-20 token based on an Ethereum UTXO custody smart contract. Unfortunately, it’s shipping time right now, so first and foremost I’m just trying to fix clarity and accuracy issues, with structure issues left for later (possibly much later...).